src/xpk/commands/info.py

```diff
@@ -0,0 +1,219 @@
+from ..core.kueue import verify_kueuectl_installation
+from .cluster import set_cluster_command
+from ..core.commands import (
+    run_command_for_value,
+)
+from ..core.core import (
+    add_zone_and_project,
+)
+import json
+from tabulate import tabulate
+from typing import Optional
+from argparse import Namespace
+
+table_fmt = 'plain'
+
+
+def prepare_kueuectl(args: Namespace) -> None:
+  """Verify if kueuectl is installed.
+  Args:
+    args: user provided arguments.
+  Returns:
+    None
+  """
+  xpk_print('Veryfing kueuectl installation')
+
+  verify_kueuectl_installed_code = verify_kueuectl_installation(args)
+  if verify_kueuectl_installed_code == 0:
+    xpk_print('kueuectl installed')
+
+  if verify_kueuectl_installed_code != 0:
+    xpk_print(
+        'kueuectl not installed. Please follow'
+        ' https://kueue.sigs.k8s.io/docs/reference/kubectl-kueue/installation/'
+        ' to install kueuectl.'
+    )
+    xpk_exit(verify_kueuectl_installed_code)
+
+
+def info(args: Namespace) -> None:
+  """Provide info about localqueues, clusterqueues and their resources.
+
+  Args:
+    args: user provided arguments for running the command.
+  Returns:
+    None
+  """
+  add_zone_and_project(args)
+
+  set_cluster_command_code = set_cluster_command(args)
+  if set_cluster_command_code != 0:
+    xpk_exit(set_cluster_command_code)
+
+  apply_shared_flags(args)
+
+  prepare_kueuectl(args)
+
+  lqs = run_kueuectl_list_localqueue(args)
+  cqs = run_kueuectl_list_clusterqueue(args)
+
+  aggregate_results(cqs, lqs)
+
+
+def apply_shared_flags(args: Namespace) -> None:
+  """Apply shared flags. It checks --project and --zone
+    flags and executes proper gcloud commands if present.
+
+  Args:
+    args: user provided args.
+
+  Returns:
+    None
+  """
+  if args.project is not None:
+    project_cmd = f'gcloud config set project {args.project}'
+    return_code, _ = run_command_for_value(project_cmd, 'Set gcp project', args)
+    if return_code != 0:
+      xpk_exit(return_code)
+
+  if args.zone is not None:
+    zone_cmd = f'gcloud config set compute/zone {args.zone}'
+    return_code, _ = run_command_for_value(zone_cmd, 'set gcloud zone', args)
+    if return_code != 0:
+      xpk_exit(return_code)
+
+
+def aggregate_results(cqs: list[dict], lqs: list[dict]) -> None:
+  """Aggregate listed clusterqueues and localqueues with resource usage and print them as table.
+
+  Args:
+    lqs: list of localqueues.
+    cqs: list of clusterqueues.
+  Returns:
+    None
+  """
+  cq_list = json.loads(cqs)['items']
+  lq_list = json.loads(lqs)['items']
+
+  cq_usages = parse_queue_lists(cq_list, usage_key='flavorsUsage')
+  lq_usages = parse_queue_lists(lq_list)
+
+  xpk_print(
+      '\n', tabulate(cq_usages + lq_usages, headers='keys', tablefmt=table_fmt)
+  )
+
+
+def parse_queue_lists(
+    qs: list[dict],
+    usage_key: Optional[str] = 'flavorUsage',
+    reservation_key: Optional[str] = 'flavorsReservation',
```

Comment:
I believe the dividend for this equation should be ClusterQueue.status.flavorsReservation and the divisor should be the nominal quota in ClusterQueue.spec